python-sdk/src/astro/extractors/facets.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, List, Optional
+
+import attr
+from openlineage.client.facet import BaseFacet
+
+from astro.constants import FileType
+from astro.table import Column, Metadata
+
+
+@attr.s
+class InputFileDatasetFacet(BaseFacet):
```

Comment:
It would be better if you used standard facets which OpenLineage consumers can properly display. However it could be ok for a first PR, it does not lock the extractors to provide this later.

Reply:
As discussed, in the case of `astro-sdk` many facets are custom and specific to our operator.